src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,117 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String NEGATIVE_INDEX_MESSAGE = "Index must be positive number";
+    private static final String INDEX_OUT_OF_BOUNDS_MESSAGE = "Index must be positive number";
+    private final ArrayList<Node<T>> data = new ArrayList<>();
```

Comment:
I don't think you need this in that task

Reply:
Why?